Source/WebCore/platform/graphics/displaylists/DisplayListItems.cpp

```diff
@@ -852,8 +852,10 @@ DrawControlPart::DrawControlPart(ControlPart& part, const FloatRoundedRect& bord
 
 void DrawControlPart::apply(GraphicsContext& context, ControlFactory& controlFactory) const
 {
-    m_part->setControlFactory(controlFactory);
+    // Temporarily override the control factory with the supplied one.
```

Comment:
This comment does not add much value. Calling `m_part->setOverrideControlFactory(nullptr)` below says the same thing.

Reply:
Correct, fixed.